LandLord-core/src/main/java/biz/princeps/landlord/placeholderapi/LLExpansion.java

```diff
@@ -42,54 +37,71 @@ public String onPlaceholderRequest(Player player, String s) {
         if (player == null) {
             return null;
         }
-        int maxClaimPermission = pl.getPlayerManager().getMaxClaimPermission(player);
-        int landcount = wg.getRegionCount(player.getUniqueId());
-        IOwnedLand region = wg.getRegion(player.getLocation());
+        final int maxClaimPermission = pl.getPlayerManager().getMaxClaimPermission(player);
+        final int landCount = wg.getRegionCount(player.getUniqueId());
+        final IOwnedLand region = wg.getRegion(player.getLocation());
 
         switch (s) {
-            case "ownedlands":
```

Comment:
also any reason to change the case of placeholders? As far as i know all placeholders are always lower case.

This would also apply to all new placeholders. We should use a consistent naming here and this seems to be all lower case

Reply:
If so, that would mean that the placeholders below do not work properly as they are case sensitive already. Other than that, this is a breaking change, it should at least keep the old option (finally a `case` where fallthrough is nice).

Reply:
Placeholders are usable as they are processed. Consistent naming is even more than using lowercases, words should be separated by a _. I’m in favor of migrating to this new format.